go/minitwit.go

```diff
@@ -88,3 +91,9 @@ func main() {
 	// Bind to a port and pass our router in
 	log.Fatal(http.ListenAndServe(":8080", r))
 }
+
+func gravatar_url(email string, size int) string {
```

Comment:
Synes vi skal bruge Go naming conventions:
https://go.dev/doc/effective_go#names

Men ved ikke om der er andrer holdninger?

Reply:
Tænker du ift. at omdøbe metoden til GetGravatar_url eller parametrene?